.github/workflows/release.yml

```diff
@@ -502,3 +503,29 @@ jobs:
           push: true
           tags: ethereumoptimism/replica-healthcheck:${{ needs.builder.outputs.replica-healthcheck }},ethereumoptimism/replica-healthcheck:latest
           build-args: BUILDER_TAG=${{ needs.builder.outputs.builder }}
+
+  batch-submitter-service:
+    name: Publish batch-submitter-service Version ${{ needs.canary-publish.outputs.canary-docker-tag }}
```

Comment:
This file has typos in it, its referring to `canary-publish` which will not exist, it should be `needs.release.outputs.batch-submitter-service`